doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -758,6 +758,9 @@ Libraries for networking
       nRF Cloud still accepts ``lng`` for backward compatibility.
     * The handling of MQTT JITP device association to improve speed and reliability.
     * To use nRF Cloud's custom MQTT topics instead of the default AWS topics.
+    * MQTT and CoAP transports to use a single unified DNS lookup mechanism that supports IPv4 and IPv6, fallback to IPv4, and handling of multiple addresses returned by getaddrinfo.
```

Comment:
Is getaddrinfo here a function, or the AT command? Regardless, it needs formatting.